cmd/tests/cmd_run_test.go

```diff
@@ -63,6 +63,35 @@ func TestSimpleTestStdin(t *testing.T) {
 	assert.Empty(t, ts.LoggerHook.Drain())
 }
 
+func TestBinaryNameStdout(t *testing.T) {
+	t.Parallel()
+
+	ts := NewGlobalTestState(t)
+	ts.BinaryName = "customBinaryName"
+	ts.CmdArgs = []string{ts.BinaryName}
+	cmd.ExecuteWithGlobalState(ts.GlobalState)
+
+	stdout := ts.Stdout.String()
+	assert.Contains(t, stdout, fmt.Sprintf("%s [command]", ts.BinaryName))
+	assert.Empty(t, ts.Stderr.Bytes())
+	assert.Empty(t, ts.LoggerHook.Drain())
+}
+
+func TestBinaryNameRunHelpStdout(t *testing.T) {
+	t.Parallel()
+
+	ts := NewGlobalTestState(t)
+	ts.BinaryName = "customBinaryName"
+	ts.CmdArgs = []string{ts.BinaryName, "help", "run"}
+	cmd.ExecuteWithGlobalState(ts.GlobalState)
+
+	stdout := ts.Stdout.String()
+	assert.Contains(t, stdout, fmt.Sprintf("%s run [flags]", ts.BinaryName))
+	assert.Contains(t, stdout, fmt.Sprintf("%s run -i 10 script.js", ts.BinaryName))
```

Comment:
Would you mind adding tests for the other commands as well? It's not a big deal, but just to be thorough.

Remember to change the test name if you do.

Reply:
No problem, done!

I did change the parent test name and did it, as you said, using table tests (didn't know about that, so thanks!).

Let me know if anything else should be done.